olm/docker-run.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+#
+# Copyright (c) 2019 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+
+# git ROOT directory used to mount filesystem
+GIT_ROOT_DIRECTORY=$(git rev-parse --show-toplevel)
+
+# Container image
+IMAGE_NAME="eclipse/che-operator-olm-build"
+
+# Operator SDK
+OPERATOR_SDK_VERSION=v0.8.2
+
+init() {
+  BLUE='\033[1;34m'
+  GREEN='\033[0;32m'
+  RED='\033[0;31m'
+  NC='\033[0m'
+  BOLD='\033[1m'
+}
+
+check() {
+  if [ $# -eq 0 ]; then
+    printf "%bError: %bNo script provided. Command is $ docker-run.sh <script-to-run> [optional-arguments-of-script-to-run]\n" "${RED}" "${NC}"
+    exit 1
+  fi
+  echo "check $1"
+  if [ ! -f "$1" ]; then
+    printf "%bError: %bscript %b provided is not existing. Command is $ docker-run.sh <script-to-run> [optional-arguments-of-script-to-run]\n" "${RED}" "${NC}" "${1}"
+    exit 1
+  fi
+}
+
+# Build image
+build() {
+  printf "%bBuilding image %b${IMAGE_NAME}${NC}..." "${BOLD}" "${BLUE}"
+  if docker build --build-arg OPERATOR_SDK_VERSION=${OPERATOR_SDK_VERSION} -t ${IMAGE_NAME} > docker-build-log 2>&1 -<<EOF
+  FROM evns/yq
```

Comment:
I don't love that we have a nested dependency on an alpine image here, since it's YET ANOTHER place where I'll have to fork this code for the product (assuming this is needed for the CRW operator?).

Could we use some variation on https://github.com/nickboldt/containers/tree/master/ubi8-yq -> https://quay.io/repository/nickboldt/ubi8-yq instead of https://github.com/evnsio/yq -> https://hub.docker.com/r/evns/yq

Reply:
but for CRW you can install stuff directly, no need to use docker wrapper

Reply:
OK, that's fair.

So... does it make sense that we start using ubi-minimal base images instead of alpine ones, in order to encourage more people to try the RHEL8 experience?

Reply:
IMHO images are too big:
```
quay.io/nickboldt/ubi8-yq                                 latest              00af3dd6c8ed        19 hours ago        148MB
mikefarah/yq                                              latest              86d84d6c94bc        2 months ago        14.9MB
evns/yq                                                   latest              a5a8e69c4986        17 months ago       54.1MB
```

Reply:
150MB to a single binary

Reply:
hmm. https://quay.io/repository/nickboldt/ubi8-yq?tab=tags says it's 50M. Does Quay lie, or are we looking at packed vs. unpacked sizes?

BTW the mikefarah version of yq is the golang implementation, not the python wrapper for `jq`, so we can't really compare that one :)

Reply:
ANYWAY if this is just something for Che 7 which productization doesn't have to worry about, then +1 for this PR.